### DISCUSSION TITLE
Show "current location" name, too.

### DISCUSSION THREAD

**Discussion by mokazemi** [breezy-weather/breezy-weather#1046] [Ideas]
### Describe your idea

It would be really better if the current location name (including city/country name) be shown on the home page top bar.

![Screenshot_20240508_185259_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/18334056/f8c1d1f3-70c9-4e58-947a-93915254c095)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.3 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Read FAQ or search existing issues, this has already been asked a trillion times

https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md

**Reply by mokazemi** [reply]
Sorry I couldn't find any related open issues in the discussions.
